events/13th-annual-users-meeting-2018.html

```diff
@@ -31,6 +31,15 @@ <h5 class="event-time">09:00 - 09:30</h5>
                 <h3 class="event-title">Morning registration with tea/coffee</h3>
             </div>
         </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">09:00</h5>
```

Comment:
ends at 09:30 ?

Reply:
Huh no, we're not doing 30min of meeting opening - especially as we talk for 2h in the afternoon :) Given the busy schedule, that was meant to be a few sentences - 5min at most. Could say `8:55 - 9:00` as this will probably be the time window where we'll move people to the conference room.

Reply:
Okay. Seems odd either way if the next talk is at 09:30.

Reply:
Oh apologies. This is meant to be 9:30